data/projects/integrate-pyflakes-AST.md

```diff
@@ -0,0 +1,21 @@
+This project focuses on integrating the [`pyflakes`](https://pypi.python.org/pypi/pyflakes)-enhanced [AST](https://en.wikipedia.org/wiki/Abstract_syntax_tree) into coala and wrap flake8 AST plugins [`flake8-future-import`](https://github.com/xZise/flake8-future-import) and [`flake8-string-format`](https://github.com/xZise/flake8-string-format) using it.
+
+This can be done by creating a meta bear like [AnnotationBear](https://github.com/coala/coala-bears/blob/1892b64722f1c10bd2a29611af4dcc18ad76af3a/bears/general/AnnotationBear.py) and [ClangBear](https://github.com/coala/coala-bears/blob/27a06d19076633a4e796472cf6d6ad55c27045c5/bears/c_languages/ClangBear.py), say PyflakesASTBear. Then use this bear to wrap the flake8 AST plugins. Tutorials for writing and testing bears can be found in our [documentation](https://api.coala.io).
+
+The flake8 plugins needed to installed can be added to a dynamically created venv. Check out [this issue](https://gitlab.com/coala/package_manager/issues/25).
+
+#### Milestones
+
+##### GSOC 2017 COMMUNITY BONDING
+
+* The participant has done prior contributions by writing and testing coala-bears. Check out [existing bear proposal](https://github.com/coala/coala-bears/issues?q=is%3Aissue+is%3Aopen+label%3A%22bear+proposal%22).
+* The participant has done prior contributions to `Pyflakes` by solving alteast one [easy task](https://bugs.launchpad.net/pyflakes/+bugs?field.tag=easy).
+
+##### GSOC 2017 MIDTERM
+
+* Integrate pyflakes-enhanced AST into coala by writing a meta bear i.e PyflakesASTBear and test the integration.
```

Comment:
Has to be completly tested and documented. Also maybe add mroe milestones to offer more points of reference